Program.cs

```diff
@@ -1,3 +1,16 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿﻿using System;
+using System.Linq;
+using System.Collections.Generic;
 
+namespace LightChess
+{
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+        }
+
+
+    }
+}
```

Comment:
All source code files should end with a new line.
https://www.quora.com/Why-is-it-a-common-practice-to-put-a-new-line-at-the-end-of-a-source-code-file


You may configure your IDE to add a finish new line automatically if missing.

Please fix in other files too.

Reply:
Fixed!

Reply:
Fixed!